src/Magnum/Platform/WindowlessEglApplication.h

```diff
@@ -313,6 +341,7 @@ class WindowlessEglContext::Configuration {
         UnsignedInt _device;
         EGLDisplay _sharedDisplay = EGL_NO_DISPLAY;
         EGLContext _sharedContext = EGL_NO_CONTEXT;
+        bool _deviceIsCuda = false;
```

Comment:
```suggestion
        UnsignedInt _device;
        UnsignedInt _cudaDevice = ~UnsignedInt{};
        EGLDisplay _sharedDisplay = EGL_NO_DISPLAY;
        EGLContext _sharedContext = EGL_NO_CONTEXT;
```

=> if `_cudaDevice` is not `~UnsignedInt{}`, it means the user set it, otherwise it'll use `_device`.